Formula/conjure-up.rb

```diff
@@ -4,7 +4,7 @@ class ConjureUp < Formula
   desc "Big software deployments so easy it's almost magical."
   homepage "http://conjure-up.io"
   url "https://github.com/conjure-up/conjure-up/archive/2.1.tar.gz"
-  sha256 "933f94c21b999fd24b4c304ba27c59bde9c83f478f5ee6a45cb8f265aaac2658"
+  sha256 "4f60e8cbbb7626c55b63928c65255cddad7df720f04ddb87f9888e1c3de1a44d"
```

Comment:
Gotcha.  In such cases it'd be better (for us at least) to have a 2.1.1 release instead of force-pushing.